translations/monero_it.ts

```diff
@@ -2386,7 +2386,7 @@ Outputs per *: </source>
     </message>
     <message>
         <location filename="../src/simplewallet/simplewallet.cpp" line="122"/>
-        <source>Create non-deterministic view and spend keys</source>
+        <source>Generate non-deterministic view and spend keys</source>
         <translation>Crea chiavi di visualizzione e chiavi di spesa non-deterministiche</translation>
```

Comment:
Are these superseded translations ever being marked stale somehow, i.e. how would a translator know which strings need to be updated?

Reply:
Using an appropriate tool for translation they do show up. One of many things on my todo-list is to document/create examples showing how this can be done. Haven't tried linguist personally because I use a tool called OmegaT for translation.

Reply:
Are you sure they will show up as »stale«? Those tools would need to scan the Git history to check whether a translation has been overhauled, right? I haven't tried any of the tools yet but somehow doubt that.

Reply:
Hold out for a little bit longer and I will publish something ...